charts/datadog-crds/templates/datadoghq.com_datadogagentprofiles_v1beta1.yaml

```diff
@@ -58,9 +58,107 @@ spec:
                       containers:
```

Comment:
do we still need to keep the v1beta1 templates?

Reply:
There were some changes in 1.8.0 so I did update and changed the script to stop updating it in the future.